packages-ts/starknet-gauntlet-ocr2/test/commands/ocr2.test.ts

```diff
@@ -1,4 +1,6 @@
 import { makeProvider } from '@chainlink/starknet-gauntlet'
+// TODO: Import from package
+import deployOZCommand from '../../../starknet-gauntlet-oz/src/commands/account/deploy'
```

Comment:
Agreed, most relative imports across packages in the monorepo should be avoided.

Reply:
I copy pasted the TODO from another test that already did this

Reply:
GH Issue: https://github.com/smartcontractkit/chainlink-starknet/issues/111